jacorb-name-server/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM jboss/base:latest
+MAINTAINER Gytis Trikleris <gytis@redhat.com>
+WORKDIR /opt/jboss
+EXPOSE 3528
+
+ENV JACORB_VERSION=2.3.1 \
+    JACORB_HOME=/opt/jboss/jacorb \
+    NAME_SERVER_PORT=3528 \
+    PATH=$PATH:/opt/jboss/jacorb/bin
+
+USER root
+
+RUN yum install -y ant \
+    && yum clean all
+
+USER jboss
+
+RUN curl -LOk http://www.jacorb.org/releases/$JACORB_VERSION/jacorb-$JACORB_VERSION-bin.zip \
```

Comment:
Hmm, that's a bit tricky then...
This JacORB name server is just a helper so maybe we could tag it with the same version as JTS i.e. 5.2.1.Final.
However, RTS and XTS are WildFly extensions so I think that makes sense to keep them in tact with WildFly version. Worst case we could tag them under the Narayana version which is in that WildFly release, but that might be a bit confusing as well. Don't you think?